linode_api4/objects/nodebalancer.py

```diff
@@ -169,6 +212,16 @@ class NodeBalancer(Base):
 
     # create derived objects
     def config_create(self, label=None, **kwargs):
```

Comment:
It seems no `label` param in the current API doc https://www.linode.com/docs/api/nodebalancers/#config-create. Not sure if we should keep it or not.

Reply:
It looks like the `NodeBalancerConfig` class doesn't have a property called `label` either so I think it should be fine to remove it.

Reply:
I agree

Reply:
Removed it.